esp-hal/src/gpio/mod.rs

```diff
@@ -170,31 +170,40 @@ pub struct RtcOutput;
 pub struct Analog;
 
 /// Drive strength (values are approximates)
-#[allow(missing_docs)]
 pub enum DriveStrength {
+    /// Drive strength of approximately 5mA.
     I5mA  = 0,
+    /// Drive strength of approximately 10mA.
     I10mA = 1,
+    /// Drive strength of approximately 20mA.
     I20mA = 2,
+    /// Drive strength of approximately 40mA.
     I40mA = 3,
 }
 
 /// Alternate functions
 #[derive(PartialEq)]
-#[allow(missing_docs)]
 pub enum AlternateFunction {
+    /// Alternate function 0.
```

Comment:
This is a case where I prefer missing documentation to whatever this is. These doc comments are supposed to be helpful, they shouldn't just echo some implementation detail. Please link to somewhere where the user can get more information about these.

Reply:
I agree, whilst we still need these mechanical comments to satisfy the missing docs lint, I'd like to see a top level doc on each enum with a bit more explanation.